ruby.js

```diff
@@ -157,6 +200,14 @@ function activate(context) {
 		balancePairs(vscode.window.activeTextEditor.document);
 	}
 
+	which("rct-complete", function(err, found) {
```

Comment:
I'm not really a fan of this call because I use `rbenv`. I'd rather it just assume it is in the path, or have a setting in the users config to point to it.

Also, this may cause a problem if the user installs it for one version of ruby, but the shell that is spawned for `which` can't find it.

I guess it's ok like this until the issues start coming in. If they don't, then all is well. 😉